.github/workflows/main.yaml

```diff
@@ -0,0 +1,20 @@
+name: Main Ci/CD
+on: [push]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      - run: echo "🔎 The name of branch is ${{ github.ref }} and repository is ${{ github.repository }}."
```

Comment:
You can write several commands like this:
run: |
    command1
    command2

Reply:
@MariaMaximova, thanks for the idea, I have updated the MR